content/kubeflow/jupyter.md

```diff
@@ -17,9 +17,17 @@ Select the namespace created in previous step:
 
 ![dashboard](/images/kubeflow/jupyter-select-namespace.png)
 
-This pre-populates the namespace field on the dashboard. Specify a name **myjupyter** for the notebook and change the CPU value to **1.0**:
+This pre-populates the namespace field on the dashboard. Specify a name **myjupyter** for the notebook:
 
-![dashboard](/images/kubeflow/jupyter-name-cpu.png)
+![dashboard](/images/kubeflow/jupyter-enter-notebook-server-name.png)
+
+In the Image section, select the **tensorflow-1.14.0-notebook-cpu** image from the dropbown box:
```

Comment:
I'd keep this generic like select latest tensorflow image for CPU. Reason being notebook images change quite rapidly.

Reply:
Fixed